modules/adfs/lib/SAML2/XML/fed/Endpoint.php

```diff
@@ -4,28 +4,29 @@
  *
  * @package SimpleSAMLphp
  */
-class sspmod_adfs_SAML2_XML_fed_Endpoint {
+class sspmod_adfs_SAML2_XML_fed_Endpoint
+{
 	/**
 	 * Add this endpoint to an XML element.
 	 *
 	 * @param DOMElement $parent  The element we should append this endpoint to.
 	 * @param string $name  The name of the element we should create.
 	 */
-	public static function appendXML(DOMElement $parent, $name, $address) {
+	public static function appendXML(DOMElement $parent, $name, $address)
+    {
 		assert('is_string($name)');
 		assert('is_string($address)');
 
 		$e = $parent->ownerDocument->createElement($name);
-                $parent->appendChild($e);
+        $parent->appendChild($e);
```

Comment:
Same here and in the next change, looks like something is odd with indentation. Maybe the rest of the code is indented wrong?